include/psa/crypto_values.h

```diff
@@ -980,6 +980,15 @@
  */
 #define PSA_ALG_XTS                             ((psa_algorithm_t)0x044000ff)
 
+/** The Electronic Code Book (ECB) mode of a block cipher, with no padding.
+ *
+ * The underlying block cipher is determined by the key type.
+ *
+ * This symmetric cipher mode can only be used with messages whose lengths
+ * are whole number of blocks for the chosen block cipher.
```

Comment:
```suggestion
 * This symmetric cipher mode can only be used with messages whose lengths
 * are a multiple of the block size of the chosen block cipher.
```

Reply:
Copied from the PSA specification. I'd rather have the specification updated before replicating any changes here.

Reply:
I've submitted a patch to change the sentence to

> This symmetric cipher mode can only be used with messages whose lengths are a multiple of the block size of the chosen block cipher.